src/sotoki/constants.py

```diff
@@ -70,10 +70,10 @@ def lang_for_domain(domain):
                 lang = get_language_details(so_code)
                 if not lang["iso-639-1"] or not lang["iso-639-3"]:
                     raise NotFound("Might be an abbreviation")
-                return lang["iso-639-1"], lang["iso-639-3"]
+                return lang["iso-639-1"], lang["iso-639-3"], True
```

Comment:
It’s not clear what this last parameter should, please document clearly the whole function